pkl-core/src/main/java/org/pkl/core/ast/member/PropertyTypeNode.java

```diff
@@ -61,9 +61,30 @@ public String getName() {
     return qualifiedPropertyName;
   }
 
+  private boolean isInIterable(VirtualFrame frame) {
+    if (frame.getArguments().length < 4) {
+      return false;
+    }
+    if (frame.getArguments()[3] instanceof Boolean isInIterable) {
+      return isInIterable;
+    }
+    return false;
+  }
```

Comment:
```suggestion
  private boolean isInIterable(VirtualFrame frame) {
    var args = frame.getArguments();
    return args.length >= 4 && args[3] instanceof Boolean b && b;
  }
```

^^ less branching ;)